### PR TITLE
Gpii 1091

### PR DESCRIPTION
**Pull request by dimokas**
http://issues.gpii.net/browse/GPII-1091

### CODE REVIEW

File: start.js [hunk 1]
```diff
@@ -11,10 +11,84 @@ https://github.com/GPII/prefsEditors/LICENSE.txt
...
+    var connect = require(kettlePath);
+    var browser = require('openurl'); // You should install openurl module. The command is: npm install openurl
+
+    connect.createServer(connect["static"](__dirname)).listen(port);
+    console.log("Preferences Management Tool server running...");
+    console.log("Visit http://"+ host + ":" + port + "/demos/prefsEditor/index.html");
+    browser.open("http://"+ host + ":" + port + "/demos/prefsEditor/index.html");
+}
+
+function findPort (callback) {
```
Comment (amb26) [inline start.js]: This workflow is unhelpful. We can't have a service running on a port which can't be predicted by the sysadmin and integrator. Please can you remove it and replace it with a simple system which accepts a port from an environment variable.

Comment (dimokas) [inline start.js]: @jobara @amb26 Fixed

File: start.js [hunk 2]
```diff
@@ -11,10 +11,22 @@ https://github.com/GPII/prefsEditors/LICENSE.txt
...
+    var connect = require(kettlePath);
+    var browser = require('openurl');
+
+    connect.createServer(connect["static"](__dirname)).listen(port);
+    console.log("Preferences Management Tool server running...");
+    console.log("Visit http://"+ host + ":" + port + "/demos/prefsEditor/index.html");
+    browser.open("http://"+ host + ":" + port + "/demos/prefsEditor/index.html");
+}
+
+startServer(port);
```
Comment (jobara) [inline start.js]: could you please add an empty new line at the end of the file?

Comment (dimokas) [inline start.js]: Done

**Comment by cstrobbe** [comment]
Just for reference: merged into review3 branch on 16 April: http://lists.gpii.net/pipermail/ux/2015-April/001333.html

**Pull request merged**